### PR TITLE
feat/DEVSU-2411-add-pto-table-custom-sort-to-printview

### PR DESCRIPTION
**Pull request by elewis2**
Adds custom row ordering for collapsible-column tables in PrintTable - ordering by evidenceLevel, gene/variant and therapy where these are available.

### CODE REVIEW

File: app/components/PrintTable/index.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { ColDef, ValueGetterParams } from '@ag-grid-community/core';
 import { v7 as createUuid } from 'uuid';
 import { registerHandlers, Handler } from 'pagedjs';
 import './index.scss';
+import { InsertEmoticon } from '@mui/icons-material';
```
Comment (bnguyen-bcgsc) [inline app/components/PrintTable/index.tsx:11]: Is this one being used?

File: app/components/PrintTable/index.tsx [hunk 2]
```diff
@@ -84,13 +89,49 @@ function PrintTable({
       const colIdxsToCombine = {};
       const rowIdxsToSkip = {};
       const rowIdxsToExpand = {};
+      const fieldList = columnDefs.map((item)=> item.field);
```
Comment (bnguyen-bcgsc) [inline app/components/PrintTable/index.tsx:92]: unused here as well

**Review by bnguyen-bcgsc — COMMENTED** [review]
- Missing some dependencies ('columnDefs', 'orderByInternalCol', and 'orderByInternalColBackup') in line 223
- Code linting

File: app/components/PrintTable/index.tsx (hunk 1 above)
Comment (elewis2) [reply app/components/PrintTable/index.tsx:11]: no! will remove

**Pull request closed without merging**